vendor/parser-shims/dune

```diff
@@ -0,0 +1,3 @@
+(library
+ (name parser_shims)
+ (libraries compiler-libs.common))
```

Comment:
Compiler-libs is only needed for the `Misc` module. We already shim that module for compatibility reasons, let's vendor `Misc`.

Reply:
Actually, vendoring `Misc` is not enough. `Misc` depends on other modules and the code for printing errors too, we'd need to vendor most of the `utils/` directory from the compiler.